filip/models/base.py

```diff
@@ -88,3 +88,13 @@ class FiwareHeader(BaseModel):
     class Config(BaseConfig):
         allow_population_by_field_name = True
         validate_assignment = True
+
```

Comment:
Almost there. Maybe we can use annotated enum here. Hence, we would alogn it with the other enums used in the library.